safety_plan.Rmd

```diff
@@ -62,7 +62,7 @@ name_pdf <- 'safety_plan'
 name_project_complete <- paste0(name_project_number, '-', name_project_name)
 
 link_repo <- paste0('https://newgraphenvironment.github.io/', name_repo, '/')
-link_kml <- paste0('https://github.com/NewGraphEnvironment/', name_repo, '/raw/main/mapping/sites_', name_project_name, '_', format(Sys.Date(), '%Y%m%d'), '_kml_gpx.zip')
+link_kml <- paste0('https://github.com/NewGraphEnvironment/', name_repo, '/raw/main/mapping')
```

Comment:
`link_kml` almost worked except I had changed the name of the file to end in '_kml.zip' vs '_kml_gpx.zip'! thats why it was failing.  

To test links that get served online you need to publish them to gitpages using the settings in your repository.  It is a bit of a pain in when doing PRs (but doable) because you will need to use settings that point to the correct branch.  Links that work locally are very commonly different than those online (ex. you need full paths to retrieve files from the net vs when you are testing locally you are reaching into your own file system.)

Been serving all downloadable files from the docs folder out of convention just to make the full paths we produce similar .

Reply:
this is interesting. I didn't know the links could be different locally vs being served on gitpages. I don't think I've published anything on gitpages yet so I will need to figure that out so that I can test the links properly.